### PR TITLE
Remove gsheet-based summing for state promote

### PR DESCRIPTION
**Pull request by breville**
Long [ago](https://github.com/code-dot-org/code-dot-org/pull/3591) we apparently took advantage of an extra row in the `cdo-state-promote` gsheet that was labeled with `"Sum_states"` rather than a state itself.  This has since caused an issue in [an iterator](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/rake/generate_pdfs.rake#L21) when we attempted to generate a PDF for a state of "Sum_states".

With this change, we just do the summing in the SQL query, rather than relying upon the sum generated by the gsheet.  This will allow us to simultaneously remove the row from the gsheet.

### CODE REVIEW

**Review by islemaster — APPROVED** [review]
:+1: Yay using our database instead!

**Review by agealy — APPROVED** [review]

**Review by uponthesun — APPROVED** [review]
I assume the traffic to the page(s) that use this is low enough that we don't need to think about query performance here? LGTM if so.

**Comment by breville** [comment]
@uponthesun Good question.  Front-ends caches these pages locally with varnish and they are cached again by the CDN, so we're pretty good on performance, but it turns out `test_pegasus_documents` also does a nice job of testing that all pegasus page-rendering database queries are cached.  I've made follow-up commits to resolve that.

**Comment by breville** [comment]
Just merged this and also removed the final row from the gsheet simultaneously.

**Pull request merged**
